homeassistant/components/tado/device_tracker.py

```diff
@@ -34,7 +35,9 @@
 )
 
 
-async def async_get_scanner(hass: HomeAssistant, config: ConfigType) -> None:
+async def async_get_scanner(
+    hass: HomeAssistant, config: ConfigType
+) -> DeviceScanner | None:
```

Comment:
This is an interface and should always have the same signature and return value type.